Angular/src/app/app.component.html

```diff
@@ -0,0 +1,160 @@
+<dx-data-grid 
+    #grid
+    [dataSource]="{ store: this.employeeStore }"
+    [showBorders]="true"
+    [repaintChangesOnly]="true"
+>
+    <dxo-editing
+        [allowUpdating]="true"
+        [allowAdding]="true"
+        [allowDeleting]="true"
+        [useIcons]="true"
+        mode="popup"
+    > </dxo-editing>
+    <dxi-column dataField="FirstName" dataType="string"></dxi-column>
+    <dxi-column dataField="LastName" dataType="string"></dxi-column>
+    <dxi-column dataField="BirthDate" dataType="date"></dxi-column>
+    <dxi-column dataField="Position" dataType="string"></dxi-column>
+    <dxi-column dataField="HireDate" dataType="date"></dxi-column>
+    <dxi-column dataField="Address" dataType="string"></dxi-column>
+    <dxi-column type="buttons">
+        <dxi-button name="edit" [onClick]="editRow"></dxi-button>
+        <dxi-button name="delete"></dxi-button>
+    </dxi-column>
+    <dxo-toolbar>
+        <dxi-item location="after">
+            <div *dxTemplate>
+                <dx-button icon="plus" (onClick)="addRow()"></dx-button>
+            </div>
+        </dxi-item>
+    </dxo-toolbar>
+</dx-data-grid>
+
+<dx-popup
```

Comment:
You can use the same approach that we used in the react:
<dx-popup @ngIf="visible"
...
visible="true"
>


https://angular.io/api/common/NgIf#description